diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -413,11 +416,11 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
                     getOrderedCallExpressions(psi, it)
                 }
             }
+            ?.takeIf { it.first() != this }
             ?.dropWhile { !it.treeParent.textContains('(') && !it.treeParent.textContains('{') }
-            // fixme: we can't distinguish fully qualified names (like java.lang) from chain of property accesses (like list.size) for now
-            ?.filter { it.getParentExpressions().count() > 1 }
-            ?.count()
-            ?.let { it > 1 }
```

Comment:
Do you still count number of statements, or will we always trigger? I think we still need to count the number of statements and issue a warning only when there are >2. No need to split very short statements.